### ISSUE TITLE
Can't download new apps

### ISSUE THREAD

**Issue by bonanza123** [mar-v-in/BlankStore#8]
I'm using the lastest blankstore build from this repo and I'm having some issues with downloading new apps (upgrading apps already listed under "installed apps" works fine).
I already removed and re-copied the APK, cleared cache and dalvik cache and removed the google account etc.

Here is a log:

```
11-08 17:09:49.159 D/SecureGooglePlay(11880): appDownload: com.cgollner.systemmonitor.lite
11-08 17:09:49.725 I/System.out(11880): GET https://android.clients.google.com/fdfe/details?doc=com.cgollner.systemmonitor.lite
11-08 17:09:49.913 I/System.out(11880): GET https://android.clients.google.com/fdfe/delivery?doc=com.cgollner.systemmonitor.lite&ot=1&vc=53
11-08 17:09:50.190 E/AndroidRuntime(11880): FATAL EXCEPTION: AsyncTask #4
11-08 17:09:50.190 E/AndroidRuntime(11880): Process: com.android.vending, PID: 11880
11-08 17:09:50.190 E/AndroidRuntime(11880): java.lang.RuntimeException: An error occured while executing doInBackground()
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at android.os.AsyncTask$3.done(AsyncTask.java:304)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at java.lang.Thread.run(Thread.java:818)
11-08 17:09:50.190 E/AndroidRuntime(11880): Caused by: java.lang.RuntimeException: No app data!
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at com.android.vending.google.SecureGooglePlayConnection.getInstallAssetSynced(SecureGooglePlayConnection.java:186)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at com.android.vending.google.SecureGooglePlayConnection.queryAppDownload(SecureGooglePlayConnection.java:211)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at com.android.vending.tasks.AppDownloadTask.doInBackground(AppDownloadTask.java:88)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at com.android.vending.tasks.AppDownloadTask.doInBackground(AppDownloadTask.java:13)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at android.os.AsyncTask$2.call(AsyncTask.java:292)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
11-08 17:09:50.190 E/AndroidRuntime(11880): 	... 4 more
```

thanks for your help in advance!

**Comment by mar-v-in** [comment]
Are you on latest version? This looks exactly like an issue that was fixed in 0.7.4.

**Comment by bonanza123** [comment]
Sorry my fault, I just copied the latest apk I found from my download folder assuming it's the latest. It's working fine. Thanks
